workflow/src/main/java/bio/terra/tanagra/workflow/WriteParentChildRelationships.java

```diff
@@ -0,0 +1,124 @@
+package bio.terra.tanagra.workflow;
```

Comment:
This is very similar to WriteAllNodes.java. Is it possible to refactor them in some way?

Reply:
They are very similar -- just a BQ read and write. I couldn't figure out how to make them use the same class because of the different schemas, though. I'll make a note to revisit this in the future, perhaps I can pass in a schema and consolidate some.